.github/workflows/mvn.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     name: Tests
     strategy:
       matrix:
-        os: [ ubuntu-20.04, macos-12 ]
+        os: [ ubuntu-20.04, macos-12, windows-latest ]
```

Comment:
@l3r8yJ can you specify a fixed version

Reply:
@h1alexbel I don't know about windows versions, which one should I set?